onnxruntime/core/session/session.cc

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifdef _WIN32
+#pragma warning(disable : 4267)
+#endif
+
+#include "core/session/session.h"
+#include "core/session/inference_session.h"
+
+using namespace ONNX_NAMESPACE;
+namespace onnxruntime {
+
+std::unique_ptr<Session> Session::Create(const SessionOptions& session_options,
+                                         logging::LoggingManager* logging_manager,
+                                         SessionType session_type) {
```

Comment:
>SessionType session_type [](start = 41, length = 24)

Just wondering whether it's better to have session_type as a field in SessionOption ?

Reply:
I'll remove the SessionType.
Training Session will have another abstract class, e.g. "class TrainableSession : public Session", and its own TrainableSession::Create() and other methods.